Week-6/Exercise6_6.js

```diff
@@ -0,0 +1,30 @@
+function closest(a, x) {
```

Comment:
This solution was deja vu for me, copied from geeksforgeeks.
copying is not an issue but did you understand well this problem and the solution?
Explain me each step in layman terms with comments and don't use comments from geeks please.